### ISSUE TITLE
Failing quickcheck test

### ISSUE THREAD

**Issue by Gabriel439** [bos/statistics#123]
I ran into the following test failure and just wanted to report it:

```
    Discrete CDF is OK: [Failed]
*** Failed! Falsifiable (after 6 tests): 
binomial 97 0.7565660170770951
N= 31    p[i]=2.0721354043840986e-20	p[i+1]=1.4030111345155002e-19	dP=1.1957975940770737e-19	relerr=1.183963930270319e-14
N= 65    p[i]=2.0389951564727095e-2	p[i+1]=3.406355217403261e-2	dP=1.3673600609305064e-2	relerr=1.3291709124073608e-14

(used seed -3743963060631687162)
```

**Comment by Gabriel439** [comment]
I forgot to mention that this is with `statistics-0.14.0.2`

**Comment by ocramz** [comment]
I can reproduce:

```
    Discrete CDF is OK: [Failed]
*** Failed! Falsifiable (after 15 tests): 
binomial 97 0.6520651223523526
N= 31    p[i]=6.62726678933811e-12	p[i+1]=2.7239842052369202e-11	dP=2.0612575263030772e-11	relerr=1.1743322589600657e-14

(used seed -142533318321840118)
```

**Comment by mightybyte** [comment]
Bump.  I'm also getting this failure.  I'm running macOS with the most recent commit (1ed1f28).

```
    Discrete CDF is OK: [Failed]
*** Failed! Falsifiable (after 18 tests):
binomial 99 0.89617703670166
N= 60    p[i]=6.883822294017256e-15     p[i+1]=3.994480899936055e-14    dP=3.306098670534285e-14        relerr=1.1217302207471003e-14

(used seed -8475207131145061897)
```

**Comment by Shimuuar** [comment]
Fixed as of e69e033d27f6d1afba0cb1b2e49a8935ed7dd4a1
